src/app/components/game-center/game-center.component.ts

```diff
@@ -18,7 +17,21 @@ export class GameCenterComponent implements OnInit {
   ) {}
 
   ngOnInit() {
-    this.openDialog();
+    this.guessWord.newRound().subscribe({
+      next: (response: any) => {
+        if (!response) {
+          this.openDialog();
+          return;
+        }
+        if (response.id) {
+          console.log(response.id);
+          return;
+        }
```

Comment:
```suggestion
        if (response?.id) {
          console.log(response.id);
        }
```

The service should answer allways, so response never will be empty. Manage error cases evaluating the fields from response (empty id, for example)